src/originWhitelist.js

```diff
@@ -39,10 +39,11 @@ const INTERNAL_BLACKLIST = [
 const EXTERNAL_WHITELIST = [
   // Google OAuth
   /\/\/accounts\.google\.com(?:$|\/)/i,
+  /\/\/www\.google\.com\/accounts\//i,
   // Google GSuite prefix sometimes used for district-specific redirects.
   /\/\/www\.google\.com\/a\//i,
   // Facebook OAuth
-  /\/\/www\.facebook\.com\/v2.6\/dialog\/oauth/i,
+  /\/\/www\.facebook\.com\/v[^/]+\/dialog\/oauth/i,
```

Comment:
While I was in here, noticed this very specific API version, and figured we should generalize it a little.